scopes/code/ui/code-compare/code-compare-view/code-compare-view.module.scss

```diff
@@ -1,10 +1,24 @@
+/**
+  * Monaco container is not flex so providing it a fixed calculated height
+  * so it can fill up the remaining space correctly instead of overflowing
+  * editor header: h4 (48px) + padding (16px) + 
+  * ignoreWhitespace (24px) + 
+  * editor padding (8px)= 96px
+  */
+$editor-padding: 96px;
+
 .componentCompareCodeViewContainer {
   width: 100%;
   height: 100%;
 }
 .componentCompareCodeDiffEditorContainer {
-  height: 100%;
-  padding: 8px 16px;
+  height: calc(100% - $editor-padding);
```

Comment:
these solutions end up causing many problems. we should try and solve with height: 100%.
you might have a `height: 100vh` above this that is causing this issue.